### PR TITLE
Fix #22

### LINKED ISSUE #22: Strange bug due to import of a lib in a Flask project
If you import the following line on a Flask project using Plugit:

```
from sqlalchemy.sql.expression import func
```

You'll end up with the following error:

```
Traceback (most recent call last):
File "server.py", line 7, in <module>
  plugit.load_actions(actions)
File "/usr/src/app/plugit/__init__.py", line 27, in load_actions
  routes.load_routes(app, act_mod)
File "/usr/src/app/plugit/routes.py", line 38, in load_routes
  PI_BASE_URL + 'meta' + obj.pi_api_route,
TypeError: cannot concatenate 'str' and '_FunctionGenerator' objects
```

Is it related to Plugit?

### PR DESCRIPTION
**Pull request by the-glu**
Fix #22 and cleaning ugly coding style

### CODE REVIEW

File: plugit/routes.py [hunk 1]
```diff
@@ -30,21 +30,21 @@ def mail():
     # (added by the decorator in utils)
     for act in dir(actions):
         obj = getattr(actions, act)
-        if hasattr(obj, 'pi_api_action') and obj.pi_api_action:
+        if hasattr(obj, 'pi_api_action') and obj.pi_api_action and not hasattr(obj.pi_api_action, '__call__'):
```
Comment (brukau) [inline plugit/routes.py]: not sure why this added, could you explain pls

Comment (the-glu) [inline plugit/routes.py]: A flask import created some function for whatever reason, with the `pi_api_action` attrubute (cf #22 ) so we just ignore them

Comment (brukau) [inline plugit/routes.py]: the issue was about `obj.pi_api_route`

Comment (the-glu) [inline plugit/routes.py]: Yes, because `pi_api_route` was missing from those objects. The condition is here to filter them

**Pull request merged**